src/main/java/com/mhcb/domain/FormConverter.java

```diff
@@ -0,0 +1,26 @@
+package com.mhcb.domain;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+import static java.util.Objects.isNull;
+
+@Converter(autoApply = true)
+public class FormConverter implements AttributeConverter<FormState, String> {
+
+    @Override
+    public String convertToDatabaseColumn(FormState state) {
```

Comment:
Could you please add the `final` keyword?

Reply:
done